cog-trt-llm/.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tensorrtllm_backend"]
+	path = tensorrtllm_backend
+	url = https://github.com/triton-inference-server/tensorrtllm_backend.git
```

Comment:
we should probably drop this

Reply:
haven't dropped it because that would break the 'legacy' dockerfile cog-trt-llm build